file.go

```diff
@@ -1,37 +1,204 @@
 package jcsv
 
-import "os"
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	data []map[string]interface{}
 }
 
 func ParseJsonFile(path string) (file, error) {
-	// TODO: open and read the given file into your `file` object
-	return file{}, nil
+	//open JSON file
+	jsonFile, err := os.Open(path)
+
+	if err != nil {
+		fmt.Println(err)
+		return file{}, err
+	}
+
+	defer jsonFile.Close()
+
+	//read complete json file and store in byteValue
+	byteValue, err := ioutil.ReadAll(jsonFile)
+
+	if err != nil {
+		fmt.Println(err)
+		return file{}, err
+	}
+
+	var myFile file
+	//convert JSON read from file in byteValue to "Data" inside file stucture
+	json.Unmarshal(byteValue, &myFile.data)
```

Comment:
You might need to handle one scenario here.
If the `.json` file contains an object instead of an array, unmarshalling into an array will cause an error.
e.g.
```json
{
	"users": [
		{
			"name" : "Ahsan"
		},
		{
			"name" : "Suleman"
		}
	]
}
```
In this case, you'll receive it in a `map[string]interface{}`, and append it to the `myFile.data`.